### PR TITLE
Doc nav image updates

### PR DESCRIPTION
**Pull request by stuartmoore1023**
PTAL - Lots here. Mostly new images with the same names. These reflect changes in the nav and layout.

### CODE REVIEW

File: jekyll/_docs/billing/updating-your-billing-info.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ You can update your billing information from
 [https://airbrake.io/billing/edit](https://airbrake.io/billing/edit) or navigate there manually
 by following these steps.
 
-### Navigate to your **Account & Billing** settings in the side bar.
+### Navigate to your **Account & billing** under the name in the upper right corner.
 ![billing settings](/docs/assets/img/docs/airbrake/billing_settings.png)
```
Comment (thompiler) [inline jekyll/_docs/billing/updating-your-billing-info.md]: Nitpick: this new image seems to be very small. Here's what it looks like in the doc:

screen shot 2017-03-27 at 12 21 01 pm

I think it would work better if it was a bit bigger.

File: jekyll/_docs/integrations/bitbucket.md [hunk 2]
```diff
@@ -29,5 +29,4 @@ step is to authenticate!
 ![bitbucket auth](/docs/assets/img/docs/integrations/bitbucket_auth.png)
 
 ### Add Bitbucket integration
-Add your Bitbucket account name and project's repository name then **save**.
-![bitbucket settings](/docs/assets/img/docs/integrations/bitbucket_settings.png)
+Log into your Bitbucket account.
```
Comment (thompiler) [inline jekyll/_docs/integrations/bitbucket.md]: This process hasn't changed. After you redirect to Bitbucket for authentication, you'll still need to specify the account name and repo name. Here's an updated screenshot you can use:
screen shot 2017-03-27 at 12 24 26 pm

File: jekyll/_docs/integrations/campfire.md [hunk 3]
```diff
@@ -47,8 +47,6 @@ the same as it appears in YOURSUBDOMAIN.campfirenow.com/rooms
 ## Testing
 Now use the **Test Integration** link from your Campfire Integration to check
 your configuration. This will send a test message from Airbrake to your
-Campfire room.
+Campfire room. You will see the message at the top if it's successful.
 ![campfire test](/docs/assets/img/docs/integrations/campfire_testing.png)
```
Comment (thompiler) [inline jekyll/_docs/integrations/campfire.md]: I haven't seen a border like this before in our app. Was this added to the image after?

screen shot 2017-03-27 at 12 28 45 pm

File: jekyll/_docs/integrations/campfire.md [hunk 4]
```diff
@@ -47,8 +47,6 @@ the same as it appears in YOURSUBDOMAIN.campfirenow.com/rooms
...
 
-You should see a message like this in your Campfire room:
-![campfire test](/docs/assets/img/docs/integrations/campfire_test_results.png)
```
Comment (thompiler) [inline jekyll/_docs/integrations/campfire.md]: Why was this removed? Does a message like that no longer get displayed?

File: jekyll/_docs/billing/updating-your-billing-info.md [hunk 5]
```diff
@@ -23,7 +23,7 @@ You can update your billing information from
 [https://airbrake.io/billing/edit](https://airbrake.io/billing/edit) or navigate there manually
 by following these steps.
 
-### Navigate to your **Account & Billing** settings in the side bar.
+### Navigate to your **Account & billing** under the name in the upper right corner.
```
Comment (phumpal) [inline jekyll/_docs/billing/updating-your-billing-info.md]: __Questions for consideration__

1. Does it make sense to remove settings? In a way these are all settings for your Airbrake account.
1. Should it read "under your name" rather "under the name"? In some cases people share a single login so it might make sense. I don't know.

File: jekyll/_docs/integrations/trello.md [hunk 6]
```diff
@@ -12,18 +12,13 @@ You can also set this integration to automatically create a Trello card when a n
...
 
 ![trello authentication](/docs/assets/img/docs/integrations/trello_auth.png)
 
 ## Select Trello board and list
-Select Trello board and Trello list where you want Airbrake to open new Trello cards.
-
-1. Select **Trello Board**
-2. Select **Trello List**
-3. Click **Save**
+Select the Trello board and the Trello list where you want Airbrake to open new Trello cards.
```
Comment (phumpal) [inline jekyll/_docs/integrations/trello.md]: Does it make sense to say  "Trello board and list"?

File: jekyll/_docs/billing/updating-your-billing-info.md (hunk 1 above)
Comment (stuartmoore1023) [inline jekyll/_docs/billing/updating-your-billing-info.md]: Done.

File: jekyll/_docs/integrations/bitbucket.md (hunk 2 above)
Comment (stuartmoore1023) [inline jekyll/_docs/integrations/bitbucket.md]: @thompiler yes, you're right. There is the "redirect for auth" step. Should that be included as well?

Comment (thompiler) [inline jekyll/_docs/integrations/bitbucket.md]: Since that's how the existing doc describes the steps, let's keep it in the same format:

screen shot 2017-03-27 at 1 15 13 pm

Comment (stuartmoore1023) [inline jekyll/_docs/integrations/bitbucket.md]: Done.

File: jekyll/_docs/integrations/campfire.md (hunk 3 above)
Comment (stuartmoore1023) [inline jekyll/_docs/integrations/campfire.md]: Done

File: jekyll/_docs/billing/updating-your-billing-info.md (hunk 5 above)
Comment (thompiler) [inline jekyll/_docs/billing/updating-your-billing-info.md]: >Does it make sense to remove settings? In a way these are all settings for your Airbrake account.

Either works but I think including the word "settings" is a little more clear.

>Should it read "under your name" rather "under the name"? In some cases people share a single login so it might make sense. I don't know.

I think "under your name" works better. I think it's okay to use language that assumes the user's name is their actual name.

Comment (mmcdaris) [inline jekyll/_docs/billing/updating-your-billing-info.md]: > Should it read "under your name" rather "under the name"?

I think that reads better.

File: jekyll/_docs/integrations/campfire.md (hunk 4 above)
Comment (stuartmoore1023) [inline jekyll/_docs/integrations/campfire.md]: Sorry. Thought that was part of the success message in Airbrake.

**Comment by mmcdaris** [comment]
Nice work @stuartmoore1023, the doc updates and new screenshots look great!

File: jekyll/_docs/billing/updating-your-billing-info.md (hunk 5 above)
Comment (stuartmoore1023) [inline jekyll/_docs/billing/updating-your-billing-info.md]: I chose "under the name" because sometimes it's not the user's name. I'm happy to change it though.

**Comment by stuartmoore1023** [comment]
PTAL

File: jekyll/_docs/integrations/trello.md [hunk 7]
```diff
@@ -12,18 +12,13 @@ You can also set this integration to automatically create a Trello card when a n
...
 ![trello authentication](/docs/assets/img/docs/integrations/trello_auth.png)
 
 ## Select Trello board and list
-Select Trello board and Trello list where you want Airbrake to open new Trello cards.
-
-1. Select **Trello Board**
-2. Select **Trello List**
-3. Click **Save**
+Select the Trello board and list where you want Airbrake to open new Trello cards.
+Select **Trello Board** then select **Trello List** and click **Save**.
```
Comment (thompiler) [inline jekyll/_docs/integrations/trello.md]: Nitpick: Any particular reason why these two numbered lists were changed to sentences?

Either option is fine but if they're sentences, I would tweak the wording a bit.

eg:
>Select the Trello board and list where you want Airbrake to open new Trello cards. Select **Trello Board** then select **Trello List** and click **Save**.

I would change to something like:

>Select the Trello board and list where you want Airbrake to open new Trello cards. Then click **save**.

Comment (stuartmoore1023) [inline jekyll/_docs/integrations/trello.md]: I didn't think the numbered steps were needed for our audience. It seemed self-explanatory.

File: jekyll/_docs/integrations/bitbucket.md [hunk 8]
```diff
@@ -29,5 +29,5 @@ step is to authenticate!
 ![bitbucket auth](/docs/assets/img/docs/integrations/bitbucket_auth.png)
 
 ### Add Bitbucket integration
-Add your Bitbucket account name and project's repository name then **save**.
-![bitbucket settings](/docs/assets/img/docs/integrations/bitbucket_settings.png)
+-Add your Bitbucket account name and project's repository name then **save**.
+ -![bitbucket settings](/docs/assets/img/docs/integrations/bitbucket_settings.png)
```
Comment (stuartmoore1023) [inline jekyll/_docs/integrations/bitbucket.md]: Missed that. Fixed.

**Comment by stuartmoore1023** [comment]
PTAL... again.

File: jekyll/_docs/integrations/trello.md (hunk 7 above)
Comment (thompiler) [inline jekyll/_docs/integrations/trello.md]: >I didn't think the numbered steps were needed for our audience. It seemed self-explanatory.

Sure that makes sense. Sounds good to me.

Let's reduce the redundancy though and condense these two sentences:
>Select the Trello board and list where you want Airbrake to open new Trello cards. Select **Trello Board** then select **Trello List** and click **Save**.

I think that could be one sentence.

File: jekyll/_docs/integrations/bitbucket.md (hunk 8 above)
Comment (thompiler) [inline jekyll/_docs/integrations/bitbucket.md]: I think the section is still there. Here's a screenshot of the redundant section:

screen shot 2017-03-28 at 11 32 09 am

Comment (stuartmoore1023) [inline jekyll/_docs/integrations/bitbucket.md]: I'm not seeing it here: https://err.io/docs/integrations/bitbucket/

screen shot 2017-03-28 at 12 04 00 pm

Comment (thompiler) [inline jekyll/_docs/integrations/bitbucket.md]: Try using an Incognito tab. This section is still in this file. It should be removed. 
screen shot 2017-03-28 at 12 10 52 pm
